website/src/_posts/2021-12-2.0-2.3.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Life after 2.0: new Audio plugin, Status Bar improvements" 
+date: 2021-12-15
+author: 
+  - aduh95
+  - murderlon
+  - arturi
+image: https://uppy.io/images/blog/2.0/uppy-2-0-cover.jpg
+published: true
+---
+
+After the 2.0 release we’ve been busy refactoring, updating dependencies, moving from npm to Yarn 3 and much-needed housekeeping. We’ve also released a new Audio plugin, improved Status Bar and Unsplash
+
+<!--more-->
+
+## Audio plugin
+
+Uppy now has a new `@uppy/audio` plugin for recording audio memos, messages and sounds. Check it out [on the demo page](https://uppy.io/examples/dashboard/).
+
+<video alt="Audio plugin demo" muted autoplay loop>
+  <source src="/images/blog/2.0-2.3/audio-demo.mp4" type="video/mp4">
+  Your browser does not support the video tag: https://uppy.io/images/blog/2.0-2.3/audio-demo.mp4
+</video>
+
+## Status bar error state improvements
+
+Status bar plugin would get confused about upload errors in Uppy, and we’ve addressed that:
+
+* Error state is shown if only _one_ failed file instead of _all_ failed files. Used to be stuck in the uploading state when a file failed.
+* Added “x of x files uploaded” below “Upload failed” for extra context.
+* Improved the error details button styling in the statusbar and the file info card.
+* Status bar state is set to complete if the user manually removes the failed files.
+
+![status bar improvements screenshot](/images/blog/2.0-2.3/status-bar-improvements.jpg)
+
+PR [#3299](https://github.com/transloadit/uppy/pull/3299).
+
+## `@uppy/unsplash` is now production ready
```

Comment:
We should make sure the prod API keys are in hosted companion before releasing this post

Reply:
That could delay the post, if the keys are already available, then yes. Can we make sure Unsplash is on uppy.io and API2 soon, @mifi?

Reply:
The keys are already available yes

Reply:
For api2, I'm thinking that because we have quite big changes (not breaking but potentially bug-introducing) from #3159 we may want to test it first on uppy.io

For uppy.io, I still haven't had time to look into this #3362

Reply:
But do we need to upgrade Companion to enable Unsplash? I think that code has already been there quite a while, so we could just add keys.

Reply:
Those are definitely needed yes.